src/app/public/modules/adapter-service/adapter.service.spec.ts

```diff
@@ -38,7 +38,7 @@ describe('Core adapter service', () => {
   describe('getFocusableChildren', () => {
     it('should return an array of all focusable children', () => {
       const actual = component.getFocusableChildren(nativeElement);
-      expect(actual.length).toEqual(6);
+      expect(actual.length).toEqual(8);
```

Comment:
Added some more unit tests to the core adapter service so we could get 100% coverage.